Neos.Neos/NodeTypes/Mixin/Document.yaml

```
@@ -7,6 +7,8 @@
     'Neos.Neos:Hidable': true
   abstract: true
   aggregate: true
+  strategy:
+    moveNode: RelationDistributionStrategy::STRATEGY_GATHER_ALL
```

```suggestion
    moveNode
```

could be `relationDistribution`

---

to make this a little easier, if we put it to options 

```yaml
options:
  relationDistributionStrategy: gatherAll
```

and access it via `Neos\ContentRepository\Core\NodeType\NodeType::getOptions`